cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -598,18 +630,16 @@ SwiftGenerator::writeServantDocSummary(IceInternal::Output& out, const Interface
     {
         out << nl << "///";
         out << nl << "/// " << name << " Methods:";
-        for (OperationList::const_iterator q = ops.begin(); q != ops.end(); ++q)
+        for (const auto& op : ops)
         {
-            OperationPtr op = *q;
-            CommentPtr opdoc = op->parseComment(swiftLinkFormatter, true);
-            out << nl << "///";
             out << nl << "///  - " << fixIdent(op->name());
+            CommentPtr opdoc = op->parseComment(swiftLinkFormatter, true);
             if (opdoc)
             {
-                out << ": ";
                 StringList opdocOverview = opdoc->overview();
                 if (!opdocOverview.empty())
                 {
+                    out << ": ";
```

Comment:
Don't emit `: ` unless it's actually going to be followed by a comment.

`opdoc` could be non-null but not contain an overview (if it only documents params/returns without an overview).